bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschSHCBridgeHandler.java

```diff
@@ -83,16 +85,21 @@ public BoschSHCBridgeHandler(Bridge bridge) {
 
     @Override
     public void initialize() {
+        logger.debug("Initialize {} Version {}", FrameworkUtil.getBundle(getClass()).getSymbolicName(),
+                FrameworkUtil.getBundle(getClass()).getVersion());
+
         // Read configuration
         BoschSHCBridgeConfiguration config = getConfigAs(BoschSHCBridgeConfiguration.class);
 
-        if (config.ipAddress.isEmpty()) {
-            this.updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "No IP address set");
+        if (config.ipAddress.trim().isEmpty()) {
+            this.updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.conf-error-empty-ip");
             return;
         }
 
-        if (config.password.isEmpty()) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "No system password set");
+        if (config.password.trim().isEmpty()) {
```

Comment:
Is it okay for the logic that uses ipAddress and password if there is empty space before or after the value? Or should we store the trimmed value in a variable and use that value later?

Reply:
I added it because I wanted to test the "conf-error" text. Afterwards I thought it would make sense to trim the IP and PW.

Yes, I should to use a local variable with the trimmed value. If the IP starts with a space we get exceptions during initial access. E.g.

    java.lang.IllegalArgumentException: Illegal character in authority at index 8: https:// 192.168.178.31:8446/smarthome/public/information